umbraco-cloud/getting-started/team-workflow-on-cloud.md

```diff
@@ -2,15 +2,15 @@
 
 In this article, we will look at some of the best practices and recommendations when you are working in teams on your Umbraco Cloud projects.
 
-## Working with GIT in Teams
+## Working with Git in Teams
 
 ### Pull before you Push
 
 Always start by making a pull request from your project before you push anything back up to Cloud. This way you ensure that you do not accidentally overwrite the work that someone else in your team has worked with.
 
 ### Create branches locally
 
-Umbraco Cloud is built on top of GIT which means that you can create branches locally as either a feature or developer branch. You can then work on the project and test out the new features before merging it into the master branch which can then be pushed up to your cloud environments.
+Umbraco Cloud is built on top of Git which means that you can create branches locally as either a feature or developer branch. You can then work on the project and test out the new features before merging it into the main branch. The branch can then be pushed up to your cloud environments.
```

Comment:
The main branch we have on Umbraco Cloud is called "master" not "main". But if you prefer you can use both: "main master branch". For example if you try to clone down a cloud project, make some changes to it, then you will see that the branch you will push to is called "master". This "master branch" is also the only one we maintain and keep on Cloud (you can read more about this [here](https://docs.umbraco.com/umbraco-cloud/getting-started/git-repository-in-a-cloud-project#:~:text=We%20only%20guarantee%20to%20maintain%20and%20keep%20the%20master%20branch.))
```suggestion
Umbraco Cloud is built on top of Git which means that you can create branches locally as either a feature or developer branch. You can then work on the project and test out the new features before merging it into the master branch. The branch can then be pushed up to your cloud environments.
```